torch/_dynamo/variables/tensor.py

```diff
@@ -788,6 +789,21 @@ def _register_hook_trampoline(tensor):
                 ),
                 **options,
             )
+            if (
+                not name.startswith("_")
```

Comment:
do we even need to check this? As long as tensor method returns itself as example value, we should probably return the original tensor regardless?